docs/release-promotion/desktop/howto_52esr_only.md

```diff
@@ -85,6 +80,52 @@ Examples
 
 ### How
 
+* **If Plan A happened for pushing to mirrors and graph expired**, you'll need to create it according sans the pushing-to-mirrors (which at this point already happened).
```

Comment:
Perhaps:

If Plan A from above "Push artifacts to releases directory" was successful but now the graph has expired, you will need to create the second graph in a similar manner to "Plan B" from above but this time we can strip the pushing-to-mirrors related tasks since that is done: